SPECS/kernel-rt/config

```diff
@@ -5626,7 +5626,7 @@ CONFIG_UIO_HV_GENERIC=m
 CONFIG_VFIO=m
 CONFIG_VFIO_IOMMU_TYPE1=m
 CONFIG_VFIO_VIRQFD=m
-# CONFIG_VFIO_NOIOMMU is not set
+CONFIG_VFIO_NOIOMMU=y
```

Comment:
Have we considered the security implications for this one? The description seems ominous 
https://cateee.net/lkddb/web-lkddb/VFIO_NOIOMMU.html#:~:text=The%20Linux%20kernel%20configuration%20item%20CONFIG_VFIO_NOIOMMU%3A%20prompt%3A%20VFIO,the%20ability%20to%20isolate%20devices%20using%20the%20IOMMU.